packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -165,12 +160,13 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
     <Controller
       name="selectedProgram"
       control={control}
-      render={({ fieldState, field: { onChange, value } }) => (
+      render={({ field: { onChange, value } }) => (
         <>
           <Select
             aria-label="program name"
             id="program"
-            invalid={!!fieldState?.error}
+            invalid={!!errors?.selectedProgram}
```

Comment:
```suggestion
            invalid={!!fieldState.error?.message}
```
The fieldState `error` is an object, and what we need to look into is the `message`, and hence the previous condition must be showing at all times

Reply:
We use `message` for the `invalidText` props. `errors` is an object that is indexable by the field name. So this works fine:

![CleanShot 2024-12-17 at 1  58 46](https://github.com/user-attachments/assets/9a05459b-250a-49f5-9f87-070a2b04fc8d)